components/home/WorkoutList.jsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/forbid-prop-types */
+import * as React from 'react';
+import {
+  FlatList, View, TouchableOpacity, Text,
+} from 'react-native';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import WorkoutContainer from './WorkoutContainer';
+
+const Title = styled.Text`
+  font-family: 'Montserrat_600SemiBold';
+  font-size: 24px;
+  margin: 0px 6%;
+  paddingBottom: 15px
+`;
+
+const WorkoutList = (props) => {
+  const { items } = props;
+
+  const renderCard = ({ item }) => (
+    <WorkoutContainer
+      name={item.name}
+      subtext={item.subtext}
+      onIconPress={item.onIconPress}
+      onPress={item.onPress}
+      color={item.color}
+    />
+  );
+
+  return (
+    <View style={{ height: '100%' }}>
+      <Title>Workouts</Title>
+      <TouchableOpacity style={{ position: 'absolute', right: 15, top: 8 }} onPress={() => alert('This should take you to the workouts page')}>
```

Comment:
Since we've got the workout page done let's hook this up so it actually takes you to the workouts page